kits/receipt-split/.env.example

```
@@ -0,0 +1,9 @@
+# Credentials for calling the deployed flows via the Lamatic API/SDK.
+# Get these from Lamatic Studio: Settings -> API Keys / Project / API Docs.
+LAMATIC_API_URL="YOUR_API_ENDPOINT"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)